src/index.html

```diff
@@ -17,11 +17,425 @@
+          <li class="nav_link">
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div
+          class="product"
+          data-qa="card"
+        >
+          <div class="product__picture product__picture--modifier"></div>
+
+          <div class="product__name product__name--modifier">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="product__code product__code--modifier">
+            Product code: 195434
+          </div>
+
+          <div class="product__rating">
+            <div class="product__rating-stars">
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+            </div>
+            <div class="product__rating-reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product__price product__price--modifier">
+            <div class="product__price-text">Price:</div>
+            <div class="product__price-digital">$2,199</div>
+          </div>
+          <div class="product__button">
+            <label
+              for="buy"
+              data-qa="card-hover"
+            >
+              <form action="#">
+                <input
+                  type="button"
+                  value="buy"
+                  id="buy"
+                  name="buy"
+                  class="product__button-buy"
+                />
+              </form>
+            </label>
```

Comment:
Avoid wrapping a 'form' element with a 'label'. Instead, use the 'for' attribute to associate the label with the input element. This is important for semantic HTML and accessibility.